app/controllers/api/distributions_controller.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+class Api::DistributionsController < Api::BaseController
+  api_versions :v1
+
+  filter_resources_by :series_id
+
+  announce_actions resource: :owner_resource
```

Comment:
Let me preface this by saying, this may be a bad idea.
When a distribution gets updated/created/deleted, I want an announcement to go out that the owner of that distribution has been changed (i.e. the series), so anything listening to changes for the series will realize there are changes to the series' distributions (oh, this is a podcast now!)